cmd/cartesi-rollups-node/main.go

```diff
@@ -43,6 +45,9 @@ func main() {
 	slog.SetDefault(logger)
 	slog.Info("Starting the Cartesi Rollups Node", "version", buildVersion, "config", config)
 
+	// run database migrations
+	repository.RunMigrations(fmt.Sprintf("%v?sslmode=disable", config.PostgresEndpoint.Value))
```

Comment:
Why `?sslmode=disable`? This doesn't seem safe.

Reply:
We need at least one issue to deal with this later before the release. Otherwise, this won't work in production.

Reply:
This PR is not blocking anything, so we can hold on merging it until the database update is done

Reply:
I've created the issue #412 to solve this problem on a later PR